testing/tests/test_custom_single_json_retriever.py

```diff
@@ -1,8 +1,72 @@
-import pytest
 import filecmp
+import json
+import perftestnotebook
+import pytest
+import yaml
+from utilities import get_nested_values
+from functools import reduce
+from operator import add
+
+
+class TestTransformer(object):
+    def default_transformer(self, test_config_filepath, target_config_filepath):
+        with open(test_config_filepath, "r") as f, open(target_config_filepath, "r") as g:
+
+            test_config = yaml.safe_load(f)
+            target_config = yaml.safe_load(g)
```

Comment:
You can get the data from these two files and immediately close the config files. That way you won't have all your other code in the indent.
```
        with open(test_config_filepath, "r") as f, open(target_config_filepath, "r") as g:
            test_config = yaml.safe_load(f)
            target_config = yaml.safe_load(g)
       assert "file_groups" in ...
```

You can do the same thing below when you open the test output.